Core/Entities/Enums.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Core.Entities
+{
+    public enum ComplaintCategory
```

Comment:
Warto dodać komentarze do tych wartości np. co oznacza `ComplaintCategory.P` albo `ComplaintCategory.SM`

Reply:
zmieniłem nazwy na Polskie nazwy własne kategorii

Reply:
Skróty były dobre, szybciej się je pisze (i IDE nie krzyczy że literówka...), wystarczyło dodać komentarze (zwykłe // przed polem, niekoniecznie te XMLowe).

Reply:
Jak już zrobiłeś to może zostać, ale możesz ocenić co lepiej wygląda - skróty + komentarze czy pełne nazwy i ewentualnie zamienić znowu